stdlib/printexc.mli

```diff
@@ -22,6 +22,11 @@ val to_string: exn -> string
 (** [Printexc.to_string e] returns a string representation of
    the exception [e]. *)
 
+val to_string_opt: exn -> string option
+(** [Printexc.to_string_opt e] returns a string option containing None if
+  there are no registered printers and Some s for a representation if there
+  are registered printers. *)
```

Comment:
Again, if it stands only: this could be more terse, in keeping with other descriptions. The type tells us that the function returns `string option` so it doesn't need stating in the description. Something more like `[Printexc.to_string_opt e] returns [None] if there are no registered printers and [Some (to_string e)] otherwise.`

Reply:
Nitpick, but I'd make the name more explicit, something like: `to_pretty_string_opt`.
(feel free to ignore this comment)